elide-core/src/main/java/com/yahoo/elide/parsers/state/RecordTerminalState.java

```diff
@@ -79,7 +79,16 @@ public RecordTerminalState(PersistentResource record, CollectionTerminalState co
         }
 
         patch(resource, state.getRequestScope());
-        return () -> Pair.of(HttpStatus.SC_NO_CONTENT, null);
+        return () -> Pair.of(
+                HttpStatus.SC_UPDATED,
+                HttpStatus.SC_UPDATED == HttpStatus.SC_NO_CONTENT
```

Comment:
When we refactor, we'll need to change the`SC_UPDATED` bit, but other than that this logic looks good to me.